src/Compilers/Core/Portable/CommandLine/AnalyzerConfig.SectionNameMatching.cs

```diff
@@ -133,7 +163,20 @@ internal SectionNameMatcher(Regex regex)
                         }
                         else
                         {
-                            // PROTOTYPE: Implement number range compilation
+                            (string numStart, string numEnd) = rangeOpt.GetValueOrDefault();
+                            if (int.TryParse(numStart, out var intStart) && int.TryParse(numEnd, out var intEnd))
+                            {
+                                var pair = intStart < intEnd ? (intStart, intEnd) : (intEnd, intStart);
+                                numberRangePairs.Add(pair);
+                                // Start the capturing group
+                                sb.Append('(');
+                                // Allow any digit sequence. The validity will be checked outside of the regex
+                                sb.Append("-?[0-9]+");
+                                // Close capturing group
+                                sb.Append(')');
```

Comment:
this seems like a constant.  could we just put this as a constant somewhere and just reference it here?